decidim-core/lib/decidim/presenter.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Decidim
+  class Presenter < SimpleDelegator
```

Comment:
Do we actually need the `Decidim::Presenter` for anything useful?

I can see that some presenters are referring to the `organization` object but I think the rest of this class is unnecessary, i.e. the delegation of missing methods to the controller's `view_context`. I also don't see the `attach_controller` being called anywhere, which is why this class looks unnecessary to me.

If the only thing this is providing is the `organization` object from the underlying object, maybe you could just delegate the `organization` method to the top-level object?

Reply:
Yes, we need that class for some of the presenters. Implementing just the SimpleDelegator, is not working ... 
The `Decidim::Admin::DashboardMetricChartsPresenter` has a call to the `content_tag` helper method.

Reply:
@ahukkanen , what i can do, is to revert the implementation, and see what are all the tests that are failing and implement the `method_missing` only for the presenters that are are failing with that particular error.

Reply:
@ahukkanen The kind of errors i get. IMHO, the cleanest solution is to have a base Presenter class to address the issues. 


- Accountability: undefined method `capture` for #<Decidim::ParticipatoryProcesses::ParticipatoryProcessMetricChartsPresenter:0x00007f21e2ac2d70>
- Admin:          undefined method `content_tag` for #<Decidim::Admin::DashboardMetricChartsPresenter:0x00007fa25d0e30b0>
- Core:           undefined method `capture` for #<Decidim::ParticipatoryProcesses::ParticipatoryProcessMetricChartsPresenter:0x00007f18f6b354c8>
- Core:           undefined local variable or method `organization` for #<Decidim::HomeStatsPresenter:0x0000556592efa1d0>
- Debates:        undefined local variable or method `organization` for #<Decidim::HomeStatsPresenter:0x000055c4029d61d0>
- Participatory:  undefined method `capture` for #<Decidim::ParticipatoryProcesses::ParticipatoryProcessMetricChartsPresenter:0x000055e714cd4de8>
- Forms:          undefined method `t` for #<Decidim::Forms::Admin::QuestionnaireParticipantPresenter:0x00005630e246f640>
- Verifications:  undefined method `content_tag` for #<Decidim::Admin::DashboardMetricChartsPresenter:0x00007f0d32d69a48>
- Proposals:      undefined method `content_tag` for #<Decidim::Admin::DashboardMetricChartsPresenter:0x000055afe1eed270>
- Surveys:        undefined method `t` for #<Decidim::Forms::Admin::QuestionnaireParticipantPresenter:0x000055cb821f8738>

Reply:
Based on this it seems the Metrics presenters only need the view context for the missing methods `capture` (`ActionView::Helpers::CaptureHelper`) and `content_tag` (`ActionView::Helpers::TagHelper`).

The missing `organization` method at `HomeStatsPresenter` could be fixed by delegating that method to the underlying object as done in the `Decidim::Presenter` class.

The missing `t` method can be fixed by replacing those with `I18n.t()` as is done in many other places around the core.

Reply:
@ahukkanen , can you have a look and tell me if this is more okay from your PoV ?

Reply:
LGTM but I added one comment regarding one added line which I believe has been a mistake.

Reply:
@ahukkanen , i have removed the addition that generated the second comment. I am going to rezolve the conversation, and let @andreslucena to finally review and merge.